databuilder/models/table_elasticsearch_document.py

```diff
@@ -26,6 +26,7 @@ def __init__(self,
         self.cluster = cluster
         self.schema_name = schema_name
         self.name = name
+        self.schema_table = '{schema}.{table}'.format(schema=schema_name, table=name)
```

Comment:
hey @jinhyukchang , Alyssa added the `display_name` field in https://github.com/lyft/amundsensearchlibrary/pull/76/files for the search service. I think we should move that model into common if we haven't and use that field for schema.table. WDYT? I was planning to do that, but haven't have time....

Reply:
Sounds good, Tao. Will make a change :)

Reply:
@feng-tao Just updated. Let me know.